src/content/app/genome-browser/components/drawer/drawer-views/variant-group-legend/VariantGroupLegend.scss

```diff
@@ -0,0 +1,47 @@
+@import 'src/styles/common';
+
+.container {
+  margin: 10px 0 0 80px;
```

Comment:
Not necessarily for this PR — but what are these magic numbers, by the way? do they want to be extracted into variables? or is this container component something we want to reuse across different drawer content?

Reply:
This could be something for design review. I am not sure about the margin. XD dev specs wasn't helpful. 
We can confirm this in the daily scrum